### PR TITLE
Added javaslang-circuitbreaker

### PR DESCRIPTION
**Pull request by RobWin**

### CODE REVIEW

**Comment by RobWin** [comment]
Not quit sure under which category you would like to put it.

File: README.md [hunk 1]
```diff
@@ -24,7 +26,7 @@ Not sure what to call these other than interoperability libraries, i.e., librari
 
 ## Reactive libraries
 
-Focused on "pulling" items from a stream. There may be overlap between these libraries and the Functional ones (above), but the ones here are primarily/exclusively focused on the [Reactive](http://www.reactive-streams.org/) way of thinking about streams.
+Focuased on "pulling" items from a stream. There may be overlap between these libraries and the Functional ones (above), but the ones here are primarily/exclusively focused on the [Reactive](http://www.reactive-streams.org/) way of thinking about streams.
```
Comment (tedyoung) [inline README.md]: Looks like a stray `a` snuck in to the word `Focused` here.

**Comment by tedyoung** [comment]
This looks like an awesome :smile: library! Seems like it'd best fit under a new section, e.g., **Networking libraries**. I know it's not exclusively aimed at networking, but it seems (?) like the most common usage. What do you think?

**Comment by vincecima** [comment]
**Patterns** as a section would work, but is probably to general. Maybe **Design Patterns** as a section for libraries that are focused on general implementations of a focused set of patterns.

**Comment by tedyoung** [comment]
I'm hesitant to have a section on patterns as they often seem to be more about learning them than having a production-ready library to use. For example, I wouldn't expect to see Hystrix listed under **(Design) Patterns**, even though it implements the Circuit Breaker pattern.

I'm thinking **Networking libraries** is sufficient. With only one new item as a candidate, I don't want to get all hung up on a taxonomy. :smile:

**Comment by vincecima** [comment]
> I'm hesitant to have a section on patterns as they often seem to be more about learning them than having a production-ready library to use.

:+1: :100:

**Comment by RobWin** [comment]
Updated chapter name and fixed typo.

**Pull request merged**